src/realm.js

```diff
@@ -179,16 +179,7 @@ const BaseRealm = {
 // where the Realm shim is loaded and executed).
 const currentUnsafeRec = createCurrentUnsafeRec();
 
-/**
- * The "primal" realm class is defined in the current "primal" environment,
- * and is part of the shim. There is no need to facade this class via evaluation
- * because both share the same intrinsics.
- */
-const Realm = buildChildRealm(currentUnsafeRec, BaseRealm);
-
-export default Realm;
-
-// factory utily for creating custom shims;
-export const factory = options => {
+// export a factory for creating "primal" realms;
+export default function(options) {
```

Comment:
Should we still export a primal `Realm` by default ?
```
// export a Realm with default config
export default Realm = makeRealmClass(); 

// export factory for creating a Realm
export function makeRealmClass(options) {};
```

Or are you thinking option 2 where we define a static `makeRealmClass` on the Realm shim itself.
```
// addition to Realm class
 class Realm {
    static makeRealmClass(options) {
         return createRealmFacade(...);
    }
}

// we still export as we did so far
export default Realm = buildChildRealm(...);


// importer is not affected
import Realm from 'realm-shim';

const RealmPrime = Realm.makeRealmClass(options); 
```

Reply:
The first. Definitely do not put a `makeRealmClass` static method on the `Realm` class. `makeRealmClass` should only be mentioned by a tiny amount of initialization code, and should never again be mentioned after initialization. We are shimming / polyfilling a spec, where the spec has a `Realm` class but no `makeRealmClass` factory function.

Reply:
And yes, I like also exporting a `Realm` class by default. For those for whom this is all they need, they would never mention the factory at all. This would be great.

The tests would then import and use this one `Realm` class.